tests/e2e/cucumber/features/smoke/app-provider/officeSuites.ocis.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Integrate with online office suites like Collabora and OnlyOffice
+  As a user
+  I want to work on different docs, sheets, slides etc.., using online office suites like Collabora or OnlyOffice
+  So that the collaboration is seamless
+
+  # To run this feature we need to run the external app-provider service along with wopi, OnlyOffice, Collabora services
+  # This is a minimal test for the integration of ocis with different online office suites like Collabora and OnlyOffice
```

Comment:
don't we need to add about these services in docs? And how about using the tag @wopi, @OnlyOffice, @Collbora to denote this feature need this services like we did for `antivirus`

Reply:
I don't think we need to have tags because we have separate pipeline to run this tests independent to of thers tests in a feature.